content/en/docs/reference/instrumentation/metrics.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Kubernetes Metrics Across Components
+content_type: instrumentation
+---
+
+
+## Metrics
+
+These are the metrics which are exported in Kubernetes components (i.e. kube-apiserver, scheduler, kube-controller-manager, kube-proxy, cloud-controller-manager). 
+
+(auto-generated 2022 Oct 25)
```

Comment:
I'd write:
```suggestion
This page details the metrics that different Kubernetes components export. You can query
the metrics endpoint for these components using an HTTP scrape, and fetch the current
metrics data in OpenMetrics format.
```
(not sure if we support OpenMetrics though).

(ideally, also add “scrape” to our glossary)

Reply:
I want to preserve the generated date so that we can know staleness, do you know where I should stick that bit?

Reply:
> I want to preserve the generated date so that we can know staleness, do you know where I should stick that bit?

How about using an HTML comment, if that's useful to maintainers.

However, SIG Docs has an existing process where we update reference docs for every major release and not (yet) for minor  releases. So the docs would normally land in a PR for a major release, and then the site generator shows the last update time details at the foot of the page.